fixtures/webversions.py

```diff
@@ -0,0 +1,48 @@
+from dotenv import load_dotenv
+import os
+import pytest
+import urllib.request
+from ast import literal_eval
+load_dotenv(verbose = True)
+url_addr_version = os.environ['URL_ADDR_VERSION']
+
+# adding cnx-deploy version number to applitools screenshot results
+@pytest.fixture(scope="session")
+def deploy_version_tag(request):
+    """Returns cnx version numbers"""
+
+    version_page = urllib.request.urlopen(url_addr_version)
+    web_content = version_page.read()
+
+    # convert bytes into strings
+    web_ver_dec = web_content.decode('UTF-8')
+    # convert strings into dictionary
+    cnx_version_dict = literal_eval(web_ver_dec)
+    # extracting cnx deploy version number
+    cnx_deploy = "-> cnx-deploy: " + cnx_version_dict['cnx-deploy']
+
+    if not cnx_deploy:
+        raise Exception("cnx deploy version is missing")
+    else:
+        yield cnx_deploy
+
+# adding webview version number to applitools screenshot results
+@pytest.fixture(scope="session")
+def webview_version_tag(request):
+    """Returns cnx version numbers"""
+
+    version_page = urllib.request.urlopen(url_addr_version)
```

Comment:
You have basically the same code in two places. This is repetitive and we don't like to repeat the same code in different places. What could you do to DRY (Don't Repeat Yourself)?